### PR TITLE
Updates for Top SQL GA

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1510,10 +1510,6 @@ set tidb_slow_log_threshold = 200;
 - 默认值：`OFF`
```
Comment (SunRunAway) [inline system-variables.md:1510]:
```suggestion
- 默认值：`ON`
```

File: dashboard/top-sql.md [hunk 2]
```diff
@@ -5,10 +5,6 @@ summary: 本文介绍如何使用 Top SQL 找到消耗负载较大的 SQL 查询
 
 # Top SQL
```
Comment (SunRunAway) [inline dashboard/top-sql.md:6]: > Top SQL 功能默认关闭。你可以通过以下方法在整个集群范围内启用该功能：

这里也要改

Comment (qiancai) [reply dashboard/top-sql.md:6]: 那 “启用 Top SQL 会对集群的性能产生轻微的影响。” 这句话还需要吗？

Comment (SunRunAway) [reply dashboard/top-sql.md:6]: 要的

Comment (SunRunAway) [reply dashboard/top-sql.md:6]: 改成 启用 / 停用 top sql吧 然后放下使用 Top SQL下面

**Review by breeswish — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: dashboard/top-sql.md [hunk 3]
```diff
@@ -5,31 +5,16 @@ summary: 本文介绍如何使用 Top SQL 找到消耗负载较大的 SQL 查询
...
-- 方法二：配置 TiDB 系统变量 [`tidb_enable_top_sql`](/system-variables.md#tidb_enable_top_sql-从-v540-版本开始引入)的值为 `ON`。
-
-> **注意：**
->
-> 启用 Top SQL 会对集群的性能产生轻微的影响。
-
 ## 使用 Top SQL
 
-开启 Top SQL 后，只需要登录 TiDB Dashboard，然后点击左侧面板中的 **Top SQL** 即可使用 Top SQL。
+Top SQL 功能默认开启。要使用该功能，只需在登录 TiDB Dashboard 后点击左侧面板中的 **Top SQL** 即可。
```
Comment (qiancai) [inline dashboard/top-sql.md:17]:
```suggestion
Top SQL 功能默认开启。要使用该功能查看消耗负载较大的 SQL 查询，只需在登录 TiDB Dashboard 后点击左侧面板中的 **Top SQL** 即可。
```

**Pull request merged**